dtl-lexer/src/tag/include.rs

```diff
@@ -0,0 +1,193 @@
+#![expect(unused_assignments)]
+use miette::{Diagnostic, SourceSpan};
+use thiserror::Error;
+
+use crate::common::text_content_at;
+use crate::tag::TagParts;
+use crate::tag::custom_tag::{
+    SimpleTagLexer, SimpleTagLexerError, SimpleTagToken, SimpleTagTokenType,
+};
+use crate::types::{At, TemplateString};
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum IncludeTemplateTokenType {
+    Text,
+    Variable,
+}
+
+#[derive(Debug, PartialEq, Eq)]
+pub struct IncludeTemplateToken {
+    pub at: (usize, usize),
```

Comment:
```suggestion
    pub at: At,
```
I suppose it should be that ?

Reply:
Something like that would work:

```yaml
  - repo: local
    hooks:
      - id: lexer-use-at
        name: "Replace `(usize, usize)` with `crate::types::At`"
        language: pygrep
        entry: \(usize, usize\)
        types: [rust]
        exclude: "dtl-lexer/src/types.rs"
```